app/views/my/access_token.html.erb

```diff
@@ -0,0 +1,140 @@
+<%#-- copyright
+OpenProject is a project management system.
+Copyright (C) 2012-2015 the OpenProject Foundation (OPF)
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License version 3.
+
+OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+Copyright (C) 2006-2013 Jean-Philippe Lang
+Copyright (C) 2010-2013 the ChiliProject Team
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License
+as published by the Free Software Foundation; either version 2
+of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+See doc/COPYRIGHT.rdoc for more details.
+
+++#%>
+
+<% html_title(l(:label_my_account), l(:label_access_token)) -%>
+
+<% breadcrumb_paths(l(:label_my_account), l(:label_access_token)) %>
+<%= toolbar title: l(:label_access_token) %>
+
+<p><%= l(:text_access_token_hint) %></p>
+
+<% if Setting.feeds_enabled != '0' ||
+      Setting.rest_api_enabled != '0' ||
+      ExtendedToken.exists?(type: 'SVNToken') %>
```

Comment:
Further thinking about it:

Why should the view have to make such hard decisions? That what controllers are for, right?

The controller could offer something like `has_tokens?` or similar (ideally a collection of token things that could be rendered by the view without further knowledge (allowing `token_things.any?)`, not sure if this is feasible though...

That would improve extensibility too:

````ruby
class FooControllerPatch
  def has_tokens?
    super || has_svn_token?
  end
end
````